decoder/gym.go

```diff
@@ -446,12 +473,18 @@ func UpdateGymRecordWithFortDetailsOutProto(db db.DbDetails, fort *pogo.FortDeta
 		gym = &Gym{}
 	}
 	gym.updateGymFromFortProto(fort)
+
+	updateGymGetMapFortCache(gym)
```

Comment:
FortDetailsOut contains name anyway doesn't it? I guess don't want to overwrite those (would need verifying that they are actually in there not just possibly in there, I can't remember)

Reply:
They were there in all cases I tested, but can do some length check if that's what you mean. You can rename fort I guess so that still would make sense.

Reply:
What I mean is that you have just received a raw which contains the name I am not sure you should overwrite it with the cached value

Reply:
Hmm.. okay make sense 👍